src/fsharp/fsi/fsi.fs

```diff
@@ -1859,20 +1861,58 @@ type internal FsiInteractionProcessor
             stopProcessingRecovery e range0    
             None
 
+    let tempDir = @"D:\temp\fsi"
+    let paketPrefix = "paket: "
+
+    let resolvePaket ctok istate errorLogger m =
+        if not needsPaketInstall then istate else
+        let paketExePath = @"D:\temp\fsi\.paket\paket.exe"
```

Comment:
Could there be a directive to specify this location before anything else?  Perhaps the default could be the local directory + .paket/paket.exe, or search directories upwards to look for a paket.exe